README.md

```diff
@@ -1,28 +1,54 @@
 # Eye-movements-rational-model-py
 
-A rational model of eye movements for identifying a single word.
+A Python implementation of a rational model of eye movements for identifying a single word.
```

Comment:
This should probably be expanded a bit to describe what this model is, and cite some papers.